server/controllers/stock/index.js

```diff
@@ -103,6 +82,60 @@ function createStock(req, res, next) {
     .done();
 }
 
+function processMovements (document, lots) {
+  
+  const items = lots || [];
+  const movements = [];
+
+  items.forEach((item) => {
```

Comment:
If you are doing a pattern like this:
```js
var x = [];

row.forEach((r) => {
  // do something
  x.push(r);
});
```
what you are really doing is a `map()`.

Your code could also be expressed  like this:
```js
function processMovements(document, lots = []) {
  const movements = lots.map((item) => {
      uuid       : db.bid(uuid.v4()),
      lot_uuid   : item.uuid,
      depot_uuid : db.bid(document.depot_uuid),
      document_uuid : db.bid(document.uuid),
      flux_id    : document.flux_id,
      date       : document.date,
      quantity   : item.quantity,
      unit_cost  : item.unit_cost,
      is_exit    : 0,
      user_id    : document.user ,
      entity_uuid : null,
      description : null
  });

  // do your filter down here to organize your code
}
```